web/src/screens/settings/Releases.tsx

```diff
@@ -66,7 +105,38 @@ function ReleaseSettings() {
               <h3 style={{ textAlign: "center" }} className="text-lg leading-6 font-medium text-gray-900 dark:text-white">
                 Danger Zone
               </h3>
-              <p style={{ textAlign: "center" }} className="mt-1 text-sm text-gray-900 dark:text-white">This will clear all release history in your database.</p>
+              <p style={{ textAlign: "center" }} className="mt-1 text-sm text-gray-900 dark:text-white">This will clear release history in your database.</p>
+            </div>
+            <div className="mt-6">
+              <label htmlFor="duration" className="items-center block text-sm font-medium text-gray-700 dark:text-white text-center">
+                Delete releases older than:
+              </label>
+              <div className="flex justify-between items-center p-2 mt-2 max-w-sm m-auto mt-1 rounded-md shadow-sm">
+                <select
+                  name="duration"
+                  id="duration"
+                  className="focus:outline-none focus:ring-2 focus:ring-offset-0 focus:ring-blue-500 dark:focus:ring-blue-500 flex-1 block w-full rounded-none rounded-r-md sm:text-sm border-gray-300 dark:border-gray-600 dark:bg-gray-800 dark:text-white"
```

Comment:
Also forgot to comment this one, the dropdown/select element will have a rounded corners only on the right side, but not on the left.